src/Components/SingleRoom/SingleRoom.css

```diff
@@ -30,7 +30,7 @@ body {
 
```

Comment:
please move global styles to separate style file `global.css` included in `App.tsx`